keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts.kt

```diff
@@ -17,46 +17,26 @@
  */
 package com.netflix.spinnaker.keel.api
 
-import com.fasterxml.jackson.annotation.JsonIgnore
-import com.fasterxml.jackson.databind.JsonDeserializer
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize
 import com.netflix.frigga.ami.AppVersion
 import com.netflix.rocket.semver.shaded.DebianVersionComparator
 import com.netflix.spinnaker.keel.api.SortType.INCREASING
 import com.netflix.spinnaker.keel.api.SortType.SEMVER
 import com.netflix.spinnaker.keel.exceptions.InvalidRegexException
-import com.netflix.spinnaker.keel.serialization.PropertyNamePolymorphicDeserializer
 import net.swiftzer.semver.SemVer
 import org.slf4j.LoggerFactory
 import org.springframework.util.comparator.NullSafeComparator
 
-/**
- * Strategy for how to sort versions of artifacts.
- */
-@JsonDeserialize(using = VersioningStrategyDeserializer::class)
-sealed class VersioningStrategy(
-  @JsonIgnore
-  open val comparator: Comparator<String>
-)
-
-@JsonDeserialize(using = JsonDeserializer.None::class)
-object DebianSemVerVersioningStrategy : VersioningStrategy(DEBIAN_VERSION_COMPARATOR) {
-  override fun toString(): String =
-    javaClass.simpleName
-
-  override fun equals(other: Any?): Boolean {
-    return other is DebianSemVerVersioningStrategy
-  }
-}
+val DockerArtifact.organization
+  get() = name.split("/").first()
```

Comment:
Made this an extension to keep the `DockerArtifact` class simple (and easier to convert to Java)